src/lib/dialog/dialog-directives.ts

```diff
@@ -0,0 +1,49 @@
+import {Directive} from '@angular/core';
+import {MdDialog} from './dialog';
+
+
+/**
+ * Button that will close the current dialog.
+ */
+@Directive({
+  selector: 'button[md-dialog-close]',
+  host: {
+    '(click)': 'dialog.closeTop()'
+  }
+})
+export class MdDialogClose {
+  constructor(public dialog: MdDialog) { }
+}
+
+/**
+ * Title of a dialog element. Stays fixed to the top of the dialog when scrolling.
+ */
+@Directive({
+  selector: '[md-dialog-title]',
+  host: {
+    role: 'heading'
+  }
+})
+export class MdDialogTitle { }
+
+
+/**
+ * Scrollable content container of a dialog.
+ */
+@Directive({
+  selector: '[md-dialog-content], md-dialog-content',
+  host: {
+    role: 'main'
```

Comment:
Shouldn't the content have some kind of role? The `main` was the only one I could find that seemed to fit.